lib/templates/fragments/comments.php

```diff
@@ -286,57 +288,67 @@ function beans_comments_navigation() {
 	}
 
 	beans_open_markup_e(
-		'beans_comments_navigation',
-		'ul',
+		'beans_comments_navigation_nav_container',
+		'nav',
 		array(
-			'class' => 'uk-pagination',
-			'role'  => 'navigation',
+			'role'       => 'navigation',
+			'aria-label' => 'Comments Pagination Navigation',
 		)
 	);
 
-		// Previous.
-	if ( get_previous_comments_link() ) {
-		beans_open_markup_e( 'beans_comments_navigation_item[_previous]', 'li', array( 'class' => 'uk-pagination-previous' ) );
+		beans_open_markup_e(
+			'beans_comments_navigation',
+			'ul',
+			array(
+				'class' => 'uk-pagination',
+			)
+		);
 
-			$previous_icon  = beans_open_markup(
-				'beans_previous_icon[_comments_navigation]',
-				'span',
-				array(
-					'class'       => 'uk-icon-angle-double-left uk-margin-small-right',
-					'aria-hidden' => 'true',
-				)
-			);
-			$previous_icon .= beans_close_markup( 'beans_previous_icon[_comments_navigation]', 'span' );
+			// Previous.
+		if ( get_previous_comments_link() ) {
+			beans_open_markup_e( 'beans_comments_navigation_item[_previous]', 'li', array( 'class' => 'uk-pagination-previous' ) );
+
+				$previous_icon  = beans_open_markup(
+					'beans_previous_icon[_comments_navigation]',
+					'span',
+					array(
+						'class'       => 'uk-icon-angle-double-left uk-margin-small-right',
+						'aria-hidden' => 'true',
+					)
+				);
+				$previous_icon .= beans_close_markup( 'beans_previous_icon[_comments_navigation]', 'span' );
 
-			echo get_previous_comments_link( // phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped -- Echoes HTML output.
-				$previous_icon . beans_output( 'beans_previous_text[_comments_navigation]', __( 'Previous Comments', 'tm-beans' ) )
-			);
+				echo get_previous_comments_link( // phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped -- Echoes HTML output.
+					$previous_icon . beans_output( 'beans_previous_text[_comments_navigation]', __( 'Previous Comments', 'tm-beans' ) )
+				);
 
-		beans_close_markup_e( 'beans_comments_navigation_item[_previous]', 'li' );
-	}
+			beans_close_markup_e( 'beans_comments_navigation_item[_previous]', 'li' );
+		}
 
-		// Next.
-	if ( get_next_comments_link() ) {
-		beans_open_markup_e( 'beans_comments_navigation_item[_next]', 'li', array( 'class' => 'uk-pagination-next' ) );
+			// Next.
```

Comment:
Shift the inline comment over one tab to line up with the `if` conditional below.